package.json

```diff
@@ -9,7 +9,8 @@
     "test-components": "eslint --fix src/**/*.js && jest tests/vue-components --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "test-c": "eslint --fix src/**/*.js && jest tests/vue-components/lookup.test.js --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
     "test-a": "eslint --fix src/**/*.js && jest tests/lib/selection/alignment --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
-    "build": "eslint --fix src/**/*.js && node --experimental-modules ./node_modules/alpheios-node-build/dist/build.mjs all all vue",
+    "build": "npm run build-safari && node --experimental-modules ./node_modules/alpheios-node-build/dist/build.mjs all all vue",
+    "build-safari": "eslint --fix src/**/*.js && node --experimental-modules ./node_modules/alpheios-node-build/dist/build.mjs all all vue-postcss config-safari.mjs",
```

Comment:
so does this mean we cannot get away from having a separate build of the components library for Safari?

Reply:
If we want to have another way of creating styles for Safari (with !important flags) - than we should use additional way of building it

Reply:
could we just add a super task, e.g. build-all which calls build and build-safari? (or better, have build-regular, build-safari and then build calls both of those

Reply:
I tried to do that - but i have such a custom message (I think it is from our node-build).
Here you could see it
![image](https://user-images.githubusercontent.com/12377640/48711481-257c1980-ec57-11e8-91e0-cbea63e46518.png)

Reply:
hmm. okay. we can figure this out later.

Reply:
And Finally - I have found my own bug in a simple line - and was able to add `buil-regular` and super build. :)
Will create a PR just now!